src/redux/mission/missionSlice.js

```diff
@@ -1,21 +1,20 @@
-/* eslint-disable no-param-reassign */
-import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
-import axios from "axios";
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
 
-const API_URL = "https://api.spacexdata.com/v3/missions";
+const API_URL = 'https://api.spacexdata.com/v3/missions';
 
 export const getMissions = createAsyncThunk(
-  "missions/getMissions",
+  'missions/getMissions',
   async (_, thunkAPI) => {
     try {
       const res = await axios(`${API_URL}`);
       return res.data;
     } catch (error) {
       return thunkAPI.rejectWithValue(
-        error?.data?.message || "Something went wrong!"
+        error?.data?.message || 'Something went wrong!',
```

Comment:
I can see that you have removed the eslint disable line as requested by the previous reviewer, however, there is an error that comes as a result. Kindly make sure there is no error. You need to work around this without disabling the linters in order to conform to the best standard practices. :+1: 

![Screenshot from 2023-05-27 21-16-03](https://github.com/LDouglasOT/space-travellers-microverse/assets/65233860/681f8749-f66d-42b2-8eaa-57289f6bfa46)